static/js/components/CandidateList.jsx

```diff
@@ -752,6 +752,16 @@ const CandidateList = () => {
             <b>Sorting: </b> Clicking on an annotation field will display it, if
             available, in the Info column. You can then click on the sort tool
             button at the top of the table to sort on that annotation field.
+            <br />
```

Comment:
Could you please fix the wording on lines 749-750 above? '"they" will be considered' or "That is, two annotation values belonging to..."

Reply:
Good catch!